app/controllers/results_controller.rb

```diff
@@ -59,8 +59,10 @@ def edit
     @first_file = @files.first
     @extra_marks_points = @result.extra_marks.points
     @extra_marks_percentage = @result.extra_marks.percentage
-    @marks_map = Hash.new
-    @old_marks_map = Hash.new
+    @flexible_marks_map = Hash.new
```

Comment:
I looked at your last comment. If we will make no distinction between flexible and rubric criteria in the summary pane, then I think I will no longer need these sets of variables. Thank you.